doc/source/whatsnew/v0.17.0.txt

```diff
@@ -397,4 +397,4 @@ Bug Fixes
 - Bug in ``io.common.get_filepath_or_buffer`` which caused reading of valid S3 files to fail if the bucket also contained keys for which the user does not have read permission (:issue:`10604`)
 - Bug in vectorised setting of timestamp columns with python ``datetime.date`` and numpy ``datetime64`` (:issue:`10408`, :issue:`10412`)
 - Bug in ``pd.DataFrame`` when constructing an empty DataFrame with a string dtype (:issue:`9428`)
-
+- Bug in ``_nanvar`` causing inaccurate results due to unstable algorithm  (:issue:`10242`)
```

Comment:
Bug in``.var()`` no-one knows (or cares how this is implemented). Be much more explict on when this occurs. E.g. highly similar values. This sounds too alarmist.